control-plane/modules/aws-k3s/files/register-to-rancher.sh

```diff
@@ -2,6 +2,7 @@
 
 %{ if is_k3s_server }
   %{ if !install_rancher }
+echo "\'${registration_command}\'"
```

Comment:
Maybe prepend this with "Registration command: ". That way if it's empty or something we know what it was supposed to be.